Core/GDCore/Project/Project.cpp

```diff
@@ -1000,6 +1005,7 @@ void Project::SerializeTo(SerializerElement& element) const {
 
   resourcesManager.SerializeTo(element.AddChild("resources"));
   SerializeObjectsTo(element.AddChild("objects"));
+  SerializeFoldersTo(element.AddChild("folderStructure"));

```

Comment:
I think that should be `objectsFolderStructure`

Reply:
Good idea